src/main/java/com/hr/jobs/controller/dto/JobDetailDto.java

```diff
@@ -0,0 +1,60 @@
+package com.hr.jobs.controller.dto;
+
+import com.hr.jobs.domain.job.Job;
+import com.hr.jobs.domain.job.SubDesc;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.Period;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Builder
+@Getter
+public class JobDetailDto {
+
+    private String companyName;
+
+    private String description;
+
+    private String city;
+
+    private List<String> subDescList;
+
+    private String createdAt;
+
+    public static JobDetailDto from(Job job) {
+        // Time
+        LocalDateTime createdAt = job.getCreatedAt();
+        LocalDateTime t2 = LocalDateTime.now();
+        Period period = Period.between(createdAt.toLocalDate(), t2.toLocalDate());
+        Duration duration = Duration.between(createdAt, t2);
+
+        String printTime = null;
+        if (duration.toMinutes() <= 60) {
+            printTime = duration.toMinutes() + " minutes ago";
+        } else if (duration.toHours() <= 24) {
+            printTime = duration.toHours() + " hours ago";
+        } else if (period.getDays() <= 31) {
+            printTime = period.getDays() + " days ago";
+        } else if (period.getMonths() <= 12) {
+            printTime = period.getMonths() + " months ago";
+        } else {
+            printTime = period.getYears() + " years ago";
+        }
```

Comment:
This line is duplicated with list API.
We can do refactoring after. (Creating one method to make the date format)